### PR TITLE
Add Trakt support to Dev

### PR DESCRIPTION
**Pull request by b3rs3rk**
Would like to get this in after NM merge if possible, team.  Please review and comment.

### CODE REVIEW

File: nzedb/libraries/TraktAPI.php [hunk 1]
```diff
@@ -12,14 +12,9 @@
 	const API_URL = 'https://api-v2launch.trakt.tv/';
 
 	/**
-	 * The Trakt.tv API v2 Client ID (SHA256 hash - 64 characters long string). Used for movie and tv lookups.
-	 * Create one here: https://trakt.tv/oauth/applications/new
-	 *
-	 * @var array|bool|string
+	 * @var array	List of site IDs that trakt,tv supports. Only trakt is guaranteed to exist.
```
Comment (niel) [inline nzedb/libraries/TraktAPI.php]: trakt,tv Typo?

**Comment by niel** [comment]
Only other comment would be more adding to the DocBlocks would be nice ;-)

File: nzedb/libraries/TraktAPI.php (hunk 1 above)
Comment (ruhllatio) [inline nzedb/libraries/TraktAPI.php]: Yep, typo.  Will fix before merge.

**Comment by ruhllatio** [comment]
For future reference @niel What additional information would you like to see?

**Comment by niel** [comment]
@ruhllatio Just basic description of what the method does, what each parameter represents, and any return value's meaning/use.
BTW this is something we all need to do better at.

**Comment by ruhllatio** [comment]
Okay.  I will keep that in mind for future work.

**Pull request merged**